packages/ui/components/AppLayout/Navbar.tsx

```diff
@@ -2,7 +2,9 @@ import React, { useState } from 'react';
 import { LetterPic } from 'components/LetterPic';
 import { Icon } from 'components/Icon';
 import { Button } from 'components/Button';
+import { Button as AntButton } from 'antd';
```

Comment:
Could we instead:

```js
import { Button as CustomButton } from 'components/Button';
import { Button } from 'antd';
```

Since we want to move away from `components/Button`. Let's do this for other imports in other files as well.